spec/std/dir_spec.cr

```diff
@@ -298,8 +297,12 @@ describe "Dir" do
       Dir[""].should eq [] of String
     end
 
-    pending_win32 "root pattern" do
-      Dir["/"].should eq ["/"]
+    it "root pattern" do
+      {% if flag?(:windows) %}
+        Dir["C:/"].should eq ["C:\\"]
+      {% else %}
+        Dir["/"].should eq ["/"]
```

Comment:
Why is it equal to a string array here??

Reply:
What else would you expect?  Path array?

Reply:
Oh, sorry, I was thinking of `Path[]`. I'm clearly tired hehe